time-it-builder/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "time-it-builder"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+lazy_static = "1.4.0"
+lazy-init = "0.5.1"
+parking_lot = "0.12.1"
```

Comment:
this seems huge. Could be good as an addition to rust hsc in places where we are forced to use sync locks, which we actually have in a few places in lua